modular_skyrat/modules/aesthetics/windows/code/windows.dm

```diff
@@ -25,3 +25,53 @@
 	canSmoothWith = list(SMOOTH_GROUP_WALLS, SMOOTH_GROUP_WINDOW_FULLTILE, SMOOTH_GROUP_AIRLOCK)
 
 /obj/structure/window/reinforced/fulltile/ice
+
+// Polarized Windows
+
+/// First defines and allows it to be assigned an ID to be toggled
+/obj/structure/window/reinforced/fulltile/polarized
```

Comment:
When talking with Funce about it, the thought flew by my head of 'oh, yeah these will be a bitch and a half to really fix or do anything' so reinforced came along as the solution

Reply:
I'm a bit confused on that given the amount of reinforced windows inside a station already, this would be an alternate to tinted as well which also share the /reinforced

Reply:
So i dont disagree with that - i do however need help on the building steps as my attempts to make it require a capacitor/cable fell flat as i couldnt get the interaction to work. I then also need to figure out how to even get people to assign a button ID to the actual window itself since these dont exist anywhere outside of machines